chatgpt/nodes/validator.py

```diff
@@ -162,16 +179,25 @@ def score_validator_performance(self, own_submission: Dict[str, Any], validator_
 
         scores = {}
 
-        for dimension in ['overall_score', 'authenticity', 'ownership', 'quality', 'uniqueness']:
+        for dimension in ['score', 'authenticity', 'ownership', 'quality', 'uniqueness']:
             if dimension in own_submission and dimension in validator_score:
-                scores[dimension] = 1 - abs(own_submission[dimension] - validator_score[dimension]) / 100
+                scores[dimension] = 1 - abs(own_submission[dimension] - validator_score[dimension] / 1e18)
```

Comment:
IMO we should apply the / 1e18 normalization step when validator_score is defined.

Reply:
Looks like validator_score is coming from the contract, so shouldn't it be already normalized?

Reply:
Maybe normalized was the wrong word, I just mean to convert to a float instead of a fixed-point integer by dividing out 1e18.